### PR TITLE
Update Carbon Ads script

### PR DESCRIPTION
**Pull request by sayzlim**
Replaced Carbon ad code with the latest version that is whitelisted
from https://adblockplus.org/. The preview for both pages can be seen
on:

Homepage: https://s.codepen.io/sayzlim/debug/dRWQPG
Docs: https://s.codepen.io/sayzlim/debug/PjmxEQ

### CODE REVIEW

**Comment by Johann-S** [comment]
Bootstrap 3 is essentially in maintenance mode as we focus on working towards a stable v4. As such, we're only accepting changes to v3's code on a case-by-case basis, usually only for critical bug fixes or docs improvements.

But we use Carbon in `v4` too, so do not hesitate to create a PR which target v4

**Comment by sayzlim** [comment]
Got it, the v4 is already using the latest Carbon ad code. We still see large amount of impressions from v3, so updating the ad code to latest version is going to receive the benefit from additional impressions.

**Comment by bardiharborow** [comment]
Pinging @mdo as this may affect the bottom line on your end.

**Comment by mdo** [comment]
This is on my list to audit and update with our next release :).

File: docs/_includes/ads.html [hunk 1]
```diff
@@ -1 +1 @@
-<div id="carbonads-container"><div class="carbonad"><div id="azcarbon"></div><script>var z = document.createElement("script"); z.async = true; z.src = "http://engine.carbonads.com/z/32341/azcarbon_2_1_0_HORIZ"; var s = document.getElementsByTagName("script")[0]; s.parentNode.insertBefore(z, s);</script></div></div>
+<script async type="text/javascript" src="//cdn.carbonads.com/carbon.js?zoneid=1673&serve=C6AILKT&placement=getbootstrapcom" id="_carbonads_js"></script>
```
Comment (XhmikosR) [inline docs/_includes/ads.html]: I'd remove the type since it's the default and also use `https` explicitly.

**Comment by mdo** [comment]
Closing as out of date. Will tackle in `gh-pages` branch.

**Pull request closed without merging**